.github/workflows/android-release-artifacts.yml

```diff
@@ -13,8 +13,24 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check-if-aar-exists:
+    name: check-if-aar-exists
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    steps:
+      - name: Check if this RC version is already in S3
+        shell: bash
+        run: |
+          VERSION="${{ inputs.version }}"
+          if curl -I "https://ossci-android.s3.amazonaws.com/executorch/release/${VERSION}/executorch.aar" | grep "200 OK"; then
```

Comment:
I'm not sure if we need to verify the checksum here too.  If yes, that can only be done after you build the aar archive as part of the linux_job workflow.  Let me know if that's the case

Reply:
The AAR checksum is not deterministic because it’s a zip and it contains zip (jar). So not quite actionable here.